trieste/acquisition/optimizer.py

```diff
@@ -113,6 +117,35 @@ def _objective() -> TensorType:
     return bijector.forward(variable)  # [1, D]
 
 
+def optimize_random(space: SearchSpace, target_func: AcquisitionFunction) -> TensorType:
+    """
+    A search across 10,000 random points :const:`AcquisitionOptimizer` defined for
+    any :class:'SearchSpace' with a :method:`sample` and batches of size of 1.
```

Comment:
To reference a specific method I think it's :meth:, not :method:, but here I think just writing method the word would be fine.